api/pkg/dataflow/service.go

```diff
@@ -42,6 +45,7 @@ const (
 	backendService_K8S     = "soda.multicloud.v1.backend"
 	s3Service_K8S          = "soda.multicloud.v1.s3"
 	dataflowService_K8S    = "soda.multicloud.v1.dataflow"
+	job_not_exist          = "job not exist"
```

Comment:
nit: Maybe you can check golang variable naming conventions. It should not have underscores.

Reply:
@subi9 Please change, 
job_not_exist to jobNotExist

Reply:
Hi Ashit/Pravin, I have changed the variable name to "jobNotExist".Please review the same.